### ISSUE TITLE
[a11y] Final breadcrumb item should be a link and use aria-current

### ISSUE THREAD

**Issue by seanes** [Altinn/altinn-components#842]
The last item in our breadcrumb trail is currently rendered as plain text. For consistency and accessibility, it should be rendered as a link and include the proper value for [aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current).

<img width="688" height="170" alt="Image" src="https://github.com/user-attachments/assets/af2311b2-c42c-47bc-9bcb-ceb2752fc028" />
